test/views/HomePageViewSpec.scala

```diff
@@ -245,14 +246,23 @@ class HomePageViewSpec extends TestSupport {
 
     "have a payment history tile" which {
       "has a heading" in new Setup {
-        getElementById("payment-history-tile").map(_.select("h2").text) shouldBe Some(messages("home.paymentHistory.heading"))
+        getElementById("payment-history-tile").map(_.select("h2").text) shouldBe Some(messages("home.paymentHistoryRefund.heading"))
       }
 
       "has a link to the payment and refund history page" in new Setup {
         val link: Option[Element] = getElementById("payment-history-tile").map(_.select("a").first)
         link.map(_.attr("href")) shouldBe Some(controllers.routes.PaymentHistoryController.show().url)
-        link.map(_.text) shouldBe Some(messages("home.paymentHistory.view"))
+        link.map(_.text) shouldBe Some(messages("home.paymentHistoryRefund.view"))
       }
+
+
+      "has an link to the 'How to claim a refund' for not migrated user" in new Setup(user = testMtdItUserNotMigrated()) {
```

Comment:
was this part of your ticket? I thought Pavel was doing something about this?

Reply:
There was comment around the test saying it should be fixed with MISUV-3609,and it was failing as it has step to test page heading.So changed it to new heading and it passed.